Dockerfile

```
@@ -0,0 +1,12 @@
+FROM node:bullseye-slim as node
+
+WORKDIR /app
+COPY . .
+RUN npm install
+RUN npm run build --prod
+
+FROM nginx:alpine
+COPY ./nginx.conf /etc/nginx/conf.d/default.conf
```

```suggestion

LABEL org.opencontainers.image.source = "https://github.com/FH-Joanneum-Iron-Road-for-Children/iron-road-for-children-web"

COPY ./nginx.conf /etc/nginx/conf.d/default.conf
```

Then the image is "connected" to this repository